index.php

```diff
@@ -145,93 +145,94 @@
             </div>
         </div>
 
+<?php
+include 'connectdb.php';
+$dishesPerPage = 12;
+$page = isset($_GET['page']) ? (int)$_GET['page'] : 1;
+$offset = ($page - 1) * $dishesPerPage;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider consolidating duplicated pagination logic</b></div>
<div id="fix">

Consider consolidating the pagination logic across files. The same pagination code (`$page` and `$offset` calculation) appears in multiple files (`index.php`, `menu.php`, and `get_dishes.php`), which could lead to maintenance issues.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
 include 'pagination_utils.php';
 calculatePagination($page, $offset, $dishesPerPage);
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Lux-Jason/rms_new/pull/6#issuecomment-2543172057>#ef655a</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged